src/logic/wallets/getWeb3.ts

```diff
@@ -116,9 +115,12 @@ export const getProviderInfo = async (web3Provider, providerName = 'Wallet') =>
   }
 }
 
-export const getAddressFromENS = async (name) => {
-  const ens = new ENS(web3)
-  return await ens.resolver(name).addr()
+export const getAddressFromENS = async (name: string) => {
+  return await web3.eth.ens.getAddress(name)
+}
+
+export const getContentFromENS = async (name: string): Promise<any> => {
```

Comment:
Should the return type be `Promise<string?>`

Reply:
@nicosampler please check this

Reply:
In this case, web3 returns a wrong [type](https://web3js.readthedocs.io/en/v1.2.8/web3-eth-ens.html?highlight=getContenthash#id61).

I'll force the type as it's specified in the docs.

Reply:
Looks like the doc is outdated https://github.com/ethereum/web3.js/blob/1.x/packages/web3-eth-ens/types/index.d.ts#L290

Reply:
<img width="653" alt="Screenshot 2020-05-28 at 16 49 21" src="https://user-images.githubusercontent.com/16622558/83143575-3555a300-a103-11ea-8873-acf4aff4758d.png">

yep, we can drop the definitions and they will be correctly inferred